.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,31 @@
+name: Unittests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Dependencies
+      run: |
+        pip install -e ".[dev]"
+        python setup.py develop
+    - name: flake8
+      run: |
+        pip install flake8
+        flake8 
+    - name: Test with pytest
+      run: |
+        pip install pytest
```

Comment:
Better run `python setup.py test`. No need to install pytest or any of the other test dependencies.